admin/src/components/HelloWorld.vue

```diff
@@ -0,0 +1,58 @@
+<template>
```

Comment:
do we need HelloWorld component here?

Reply:
It used on Home page, but I've removed it